### ISSUE TITLE
fastdtw as SVM kernel

### ISSUE THREAD

**Issue by mohsin671** [slaypni/fastdtw#24]
Can I use this fastdtw as a custom kernel to SKlearn SVM classifier?

**Comment by slaypni** [comment]
I do not think it is valid to use DTW as a kernel function, although it could return a value.

**Comment by slaypni** [comment]
I found a paper by googling. Although I did not read it, it might be informative.

Wan, V., & Carmichael, J. (2005). Polynomial dynamic time warping kernel support vector machines for dysarthric speech recognition with sparse training data. In Ninth European Conference on Speech Communication and Technology.
